qiita_ware/ebi.py

```diff
@@ -105,21 +74,36 @@ class EBISubmission(object):
     Parameters
     ----------
     preprocessed_data_id : str
+
+    Raises
+    ------
+    EBISumbissionError
+        If either the required columns for EBI and Qiita deploy are not
+        present in the sample and prep templates
```

Comment:
It seems like there's a few missing reasons why this error can be raised, would you mind listing them?